src/chunkserver/copyset_node.cpp

```diff
@@ -658,6 +665,11 @@ bool CopysetNode::IsLeaderTerm() const {
     return false;
 }
 
+bool CopysetNode::IsLeaseLeader(const braft::LeaderLeaseStatus &lease_status) const {  // NOLINT
+    auto term = leaderTerm_.load(std::memory_order_acquire);
+    return !(term < 0 || term != lease_status.term || lease_status.state == braft::LEASE_EXPIRED);  // NOLINT
```

Comment:
```cpp
return term == lease_status.term && (lease_status.state == LEASE_VLIAD || lease_status.state == LEASE_NOT_READ)
```

Reply:
if lease state is `LEASE_NOT_READ`, maybe term is different.(lease change status, but `on_leader_start` doesn't run in state machine).

Reply:
this method seems is better~
```CPP
bool CopysetNode::IsLeaseLeader(const braft::LeaderLeaseStatus &lease_status) const {  // NOLINT
    auto term = leaderTerm_.load(std::memory_order_acquire);
    return lease_status.state == LEASE_NOT_READ ||
            (term > 0 && term == lease_status.term && lease_status.state == LEASE_VLIAD);  // NOLINT
}
```

Reply:
It seems that we judge `LEASE_NOT_READY` is meaningless.
if `term != lease_status.term`, it means that `on_leader_start` have run in state machine, so the lease status must be `LEASE_VLIAD ` in this condition(and we will do local-read in current state machine).
I think we ought to judge lease state in following code(if lease state is `LEASE_NOT_READY`, we will do log-read in this condition):
https://github.com/opencurve/curve/blob/4503063cf46a1f9f3a238252a34dd53a8284e060/src/chunkserver/op_request.cpp#L253-L256